src/phoenix/server/api/types/Span.py

```diff
@@ -89,7 +89,7 @@ def from_mapping(
         """Converts a mapping to a SpanEvent. Used when parsing the record from the dataframe."""
         return SpanEvent(
             name=mapping["name"],
-            message=mapping["message"],
+            message=(mapping.get("attributes") or {}).get(sc.EXCEPTION_MESSAGE) or "",
```

Comment:
```suggestion
            message=mapping.get("attributes", {}).get(sc.EXCEPTION_MESSAGE, ""),
```

Suggestion, non-blocking.

Reply:
this would break if "attributes" somehow exists and has `None` as value

<img width="861" alt="Screenshot 2023-08-29 at 6 08 36 PM" src="https://github.com/Arize-ai/phoenix/assets/80478925/f70f8f49-03f2-40f2-9573-f9b5c0014c05">